### PR TITLE
Remove unused bridge code

### PR DESCRIPTION
**Pull request by Mrtenz**
Needs to be rebased once #90 is merged.

### CODE REVIEW

**Comment by Mrtenz** [comment]
[ch8498]

File: src/app/screens/__tests__/AddAccount.test.tsx [hunk 1]
```diff
@@ -293,106 +304,18 @@ describe('AddAccount', () => {
...
     });
 
-    const submitButton = getByText(translateRaw('NEXT'));
-    expect(submitButton).toBeDefined();
-    fireEvent.click(submitButton);
-
-    await waitFor(() => expect(getByText('error', { exact: false })).toBeDefined());
-  });
-
-  it('can submit mnemonic from another DPath and page', async () => {
```
Comment (FrederikBolding) [inline src/app/screens/__tests__/AddAccount.test.tsx:321]: I would prefer if we could keep this test. Would it be possible to run it with a regular store to fix the problems you've had?

File: src/app/screens/AddAccount/AddAccountMnemonic.tsx [hunk 2]
```diff
@@ -100,6 +111,7 @@ const AddAccountMnemonicForm = ({
   };
 
   const handleSubmit = async () => {
+    dispatch(setAddresses([]));
```
Comment (FrederikBolding) [inline src/app/screens/AddAccount/AddAccountMnemonic.tsx:114]: Is this needed? Doesn't fetch addresses overwrite?

File: src/bridge.spec.ts [hunk 3]
```diff
@@ -1,50 +1,73 @@
...
-      type: CryptoRequestType.GET_ADDRESS,
-      wallet: {
-        walletType: WalletType.PRIVATE_KEY,
-        privateKey: '0x93b3701cf8eeb6f7d3b22211c691734f24816a02efa933f67f34d37053182577'
-      }
-    };
-    IpcBridgeRenderer(mockIpcRenderer).crypto.invoke(request);
-    expect(mockIpcRenderer.invoke).toHaveBeenCalledWith(IPC_CHANNELS.CRYPTO, request);
+  describe('on', () => {
+    it('listens to an event', () => {
```
Comment (FrederikBolding) [inline src/bridge.spec.ts:62]: Perhaps add the unsubscribe test as part of the text?

**Review by FrederikBolding — CHANGES_REQUESTED** [review]

**Review by FrederikBolding — APPROVED** [review]

**Pull request merged**